Library/Formula/mahout.rb

```diff
@@ -30,6 +30,7 @@ def install
     end
 
     bin.write_exec_script Dir["#{libexec}/bin/*"]
```

Comment:
You should delete this line. It's covered by line in below. 

Also can you remove `export JAVA_HOME` in caveats? thanks.